index.ejs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
By default the [html-webpack-plugin](https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates) uses the [lodash template loader](https://github.com/okonet/ejs-loader). This doesn't really impact us much - and we can change the naming scheme. Ultimately i'd like to combine index.ejs and index.html - but need to do a bit more research as to why the plugin wasn't working in dev.